navylinux/content.md

```diff
@@ -0,0 +1,86 @@
+# Navy Linux Enterprise
+
+Navy Linux Enterprise is an open-source community-powered enterprise Linux operating system is a rebuild of the Red Hat Linux project just as the CentOS Linux project has been for all these years. Navy Foundation that based on the open standards to build Navy Linux Project in consonance with open source software development standards. Act as code to keep technologies open, usable for productive operation..
+
+We promise the same experience that was the hallmark of the CentOS Linux
+
+> [Navy Linux Wiki](https://navylinux.org/wiki/)
+
+%%LOGO%%
+
+# Navy Linux image documentation
+
+The `%%IMAGE%%:latest` tag is always the most recent version currently available.
+
+## Rolling builds
+
+The Navy Linux Project offers regularly updated images for all active releases. These images will be updated monthly or as needed for emergency fixes. These rolling updates are tagged with the major version number only. For example: `docker pull %%IMAGE%%:8`
+
+## Overlayfs and yum
+
+Recent Docker versions support the [overlayfs](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/) backend, which is enabled by default on most distros supporting it from Docker 1.13 onwards.
```

Comment:
There's not any version of Navy Linux that has issues with this, right?  So this whole section should probably go away:

```suggestion
```